app/helpers/collection_helper.rb

```diff
@@ -1,4 +1,9 @@
 module CollectionHelper
+  # strip leading 'a' from collection catkeys
```

Comment:
Especially with the string-based implementation and given the very clear name you've given the method, I think we could also consider dropping the comment.